app/scripts/lib/experiments/grouping-rules/email-first.js

```diff
@@ -75,17 +77,6 @@ define((require, exports, module) => {
     static sampleRate (env) {
       return env === 'development' ? 1.0 : 0.2;
     }
-
-    /**
-     * The experiment name
-     *
-     * @readonly
-     * @static
-     * @return {String}
-     */
-    static get name () {
```

Comment:
👍

Reply:
Nice! I'll update iOS bug once this gets tagged.